src/commands/GetInfoCommand.cpp

```diff
@@ -622,6 +628,29 @@ bool GetInfoCommand::SendLabels(const CommandContext &context)
    return true;
 }
 
+bool GetInfoCommand::SendTime(const CommandContext &context)
```

Comment:
I think this can be refactored like this:

```
bool GetInfoCommand::SendTime(const CommandContext &context)
{
    context.StartArray();

    const auto& project = context.project;
    
    // Check if audio is active and get the appropriate time
    const double audioTime = ProjectAudioIO::Get(project).IsAudioActive()
        ? (AudioIO::Get() ? AudioIO::Get()->GetStreamTime() : 0.0)
        : ViewInfo::Get(project).playRegion.GetStart();

    // Send cursor position
    context.AddItem(audioTime, "CurrentPosition");

    context.EndArray();

    return true;
}
```